### ISSUE TITLE
API /api/incidents not working

### ISSUE THREAD

**Issue by rohitpmore** [bosun-monitor/bosun#1820]
Hi All, 
From the code (0.5.0), it seems that "/api/incidents" code is not yet added, which was available in previous releases.
Is it going to be implemented in future?

Thanks,
Rohit.

**Comment by TrentScholl** [comment]
If you're trying to get a list of the current active incidents, have a look at /api/alerts

**Comment by lukaspustina** [comment]
Works fine for me with released binary version 0.5.0 in case of open incidents -- see below. You can get more information about a specific incidents, e.g. 1, via `/api/incidents/events?id=1`.

```
[7:11:49 lukas@muffin] ~  ➜  curl -H 'Accept: application/json' -X GET http://192.168.99.100:8070/api/incidents/open | jq '.'
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100   720  100   720    0     0   216k      0 --:--:-- --:--:-- --:--:--  351k
[
  {
    "Id": 1,
    "Subject": "warning: <no value> on cake",
    "Start": 1467975766,
    "AlertName": "test.lukas",
    "Tags": {
      "host": "cake"
    },
    "TagsString": "{host=cake}",
    "CurrentStatus": "normal",
    "WorstStatus": "warning",
    "LastAbnormalStatus": "warning",
    "LastAbnormalTime": 1468330514,
    "Unevaluated": false,
    "NeedAck": false,
    "Silenced": false,
    "Actions": [
      {
        "User": "lukas.pustina",
        "Message": "I don’t care",
        "Time": 1468326958,
        "Type": "Acknowledged"
      }
    ],
    "Events": [
      {
        "Status": "warning",
        "Time": 1467975737
      },
      {
        "Status": "normal",
        "Time": 1467979392
      },
      {
        "Status": "warning",
        "Time": 1468219885
      },
      {
        "Status": "normal",
        "Time": 1468223515
      },
      {
        "Status": "warning",
        "Time": 1468326944
      },
      {
        "Status": "normal",
        "Time": 1468330604
      }
    ],
    "WarnNotificationChains": [],
    "CritNotificationChains": []
  }
]
```

**Comment by rohitpmore** [comment]
@lukaspustina Yes, there was one more api before "/api/incidents" whose code is commented out in the release 0.5.0 with "TODO: Incident Search". 
@TrentScholl Yes, I am using Bosun API's including /api/alerts for my own dashboards and I was using /api/incidents as well.

Thanks,
Rohit.
